debian/changelog

```diff
@@ -1,3 +1,10 @@
+ubuntu-core-initramfs (66.1) UNRELEASED; urgency=medium
```

Comment:
Won't rev 66.1 cause trouble when uploading? 67 was reverted, but even in that case.